CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ if(WIN32)
   # https://github.com/msys2/MINGW-packages/issues/3319
   add_definitions(
     -D_WIN32_WINNT=${WIN32_WINNT}
+    -DFMT_USE_TZSET=0
```

Comment:
If this is done globally now, is the existing `$<$<BOOL:${WIN32}>:FMT_USE_TZSET=0>` incantation still needed?

Reply:
Right, I think we can go ahead and drop it.